concepticondata/conceptlists/Kraft-1981-434.tsv

```diff
@@ -334,7 +334,7 @@ Kraft-1981-434-332	1595	SOUTH	332	kudù	south
 Kraft-1981-434-333	1877	EAST	333	gabàs	east
 Kraft-1981-434-334	1817	WEST	334	yâmma	west
 Kraft-1981-434-335	2152	THERE IS OR THERE ARE	335	àkwai àbinci	there is
-Kraft-1981-434-336	1240	NOT	336	bābù àbinci	there is not
+Kraft-1981-434-336			336	bābù àbinci	there is not
```

Comment:
I would be inclined to use: https://concepticon.clld.org/parameters/2329
As we link similar expressions in SEA languages to this.

Reply:
Oh, didn't notice we have that concept. Seems close enough.